Cargo.toml

```diff
@@ -1,6 +1,6 @@
 [package]
 name = "l337"
-version = "0.9.0"
+version = "0.10.0"
```

Comment:
If `l337` version is updated then the `l337` dependencies in the impl crates also need to change

Reply:
I'll squash those commits when I merge.